joplin/base/management/commands/load_joplin_data.py

```diff
@@ -89,18 +89,27 @@ def load_fixture(operation_name, fixture_name, condition):
             load_fixture(
                 "load_test_admin",
                 'db/fixtures/local_admin_user.json',
-                (os.getenv("DEPLOYMENT_MODE") == "LOCAL")
+                (
+                    os.getenv("DEPLOYMENT_MODE") == "LOCAL"
+                    or settings.V3_WIP # if we don't LOAD_DATA in v3, we still need to add a test admin user
```

Comment:
oh cool! I've been manually `createsuperuser`ing in heroku console

Reply:
I didn't know that was an option! Cool. Anyway, its an automated option now.

Reply:
Oh cool, I didn't know that was an option. Anyways, its automated now.